deeplabcut/utils/visualization.py

```diff
@@ -394,6 +444,8 @@ def plot_evaluation_results(
     dot_size: int = 12,
     alpha_value: float = 0.7,
     p_cutoff: float = 0.6,
+    bounding_boxes: dict = {},
```

Comment:
A `dict` shouldn't be used as a default argument as it's mutable. Should set:

```python
def make_multianimal_labeled_image(
    ...
    bounding_boxes: dict | None = None,
    ...
) -> plt.Axes:
    if bounding_boxes is None:
        bounding_boxes = {}
```

Reply:
Right!